pages/osx/airport.md

```diff
@@ -5,16 +5,16 @@
 
 - Show current wireless status information:
 
-`airport -I`
+`airport --getinfo`
 
 - Sniff wireless traffic on channel 1:
 
 `airport sniff {{1}}`
 
 - Scan for available wireless networks:
 
-`airport -s`
+`airport --scan=`
```

Comment:
what about that `=`?

Reply:
It is [intended](https://ss64.com/osx/airport.html):

```
      -s[arg]
      --scan=[arg]     Perform a wireless broadcast scan.
                       Will perform a directed scan if the optional arg is provided.
```

as I figured out from docs.

Reply:
I'm sure is not required, if no `arg` is provided
```suggestion
`airport --scan`
```